HowToKerberize.md

```diff
@@ -0,0 +1,106 @@
+![Bee waggle-dancing on a hive.](logo.png "Federating Hive Meta Stores.")
+
+# Additional instruction to use Waggle Dance in kerberized environment
+
+
+### Process
+
+In a kerberos environment, after a client requests waggle-dance, waggle-dance requests the proxy user's token from the metastore and then uses that token to communicate with the metastore.
+
+This is necessary in some scenarios that require permission authentication. Such as the `create_table` API that requires the proxy user to create hdfs directories.
+
+![image](https://user-images.githubusercontent.com/13965087/229339323-260f3c17-46c0-4471-81d2-cdbcfa0fe3ce.png)
```

Comment:
We should move a cleaned up version of this diagram to this repository instead of an external link.

Reply:
I think what @massdosage means is to upload the image as part of the PR and reference it just like we do with: https://github.com/ExpediaGroup/waggle-dance/blob/main/system-diagram.png